src/qibolab/instruments/emulator/backends/generic.py

```diff
@@ -0,0 +1,315 @@
+        latex_op_dict["bdag"].update({k: rf"b^{{\dagger}}_{k}"})
+        latex_op_dict["O"].update({k: f"O_{k}"})
+        latex_op_dict["X"].update({k: f"X_{k}"})
+        latex_op_dict["Z01"].update({k: rf"\sigma^Z_{k}"})
+        latex_op_dict["sp01"].update({k: rf"\sigma^+_{k}"})
+
+    latex_op_connectors_dict = OrderedDict(
+        [
+            ("^", lambda a, b: rf"{a}{{\otimes}}{b}"),
+            ("*", lambda a, b: f"{a}{b}"),
+            ("+", lambda a, b: f"{a}+{b}"),
+            ("-", lambda a, b: f"{a}-{b}"),
+        ]
+    )
+
+    basic_dict = [
+        rf"$O_i = b^{{\dagger}}_i b_i$",
+        rf"$X_i = b^{{\dagger}}_i + b_i$",
+    ]
+    print("Dictionary")
+    for i in basic_dict:
+        display(Latex(i))
+    print("\n")
+    print("-" * 21)
+
+    print("One-body drift terms:")
+    print("-" * 21)
+    for op_instruction in model_config["drift"]["one_body"]:
+        op_instruction = op_from_instruction(
+            op_instruction,
+            latex_op_dict,
+            latex_op_connectors_dict,
+            multiply_coeff=False,
+        )
+        inst = format_instruction(op_instruction)
+        display(Latex(inst[0]))
+        display(Latex(inst[1]))
+    print("-" * 21)
+
+    print("Two-body drift terms:")
+    print("-" * 21)
+    if len(model_config["drift"]["two_body"]) == 0:
+        print("None")
+    for op_instruction in model_config["drift"]["two_body"]:
+        op_instruction = op_from_instruction(
+            op_instruction,
+            latex_op_dict,
+            latex_op_connectors_dict,
+            multiply_coeff=False,
+        )
+        inst = format_instruction(op_instruction)
+        display(Latex(inst[0]))
+        display(Latex(inst[1]))
+    print("-" * 21)
+
+    print("One-body drive terms:")
+    print("-" * 21)
+    for drive_instructions in model_config["drive"].values():
+        for op_instruction in drive_instructions:
+            op_instruction = op_from_instruction(
+                op_instruction,
+                latex_op_dict,
+                latex_op_connectors_dict,
+                multiply_coeff=False,
+            )
+            inst = format_instruction(op_instruction)
+            display(Latex(inst[0]))
+            display(Latex(inst[1]))
+    print("-" * 21)
+
+    print("Dissipative terms:")
+    print("-" * 21)
+    for key in model_config["dissipation"].keys():
+        print(">>", key, "Linblad operators:")
+        if len(model_config["dissipation"][key]) == 0:
+            print("None")
+        for op_instruction in model_config["dissipation"][key]:
+            op_instruction = op_from_instruction(
+                op_instruction,
+                latex_op_dict,
+                latex_op_connectors_dict,
+                multiply_coeff=False,
+            )
+            inst = format_instruction(op_instruction, is_dissipation=True)
+            display(Latex(inst[0]))
+            display(Latex(inst[1]))
+    print("-" * 21)
```

Comment:
This function is specific for a Jupyter Notebook environment, as it is possible to infer from the `display()` and `Latex()` usage.

Moreover, this is an interface function, but Qibolab is mostly a backend handling execution (so it should not expose a UI, since users will almost never interact directly with it).

I'd propose moving this function somewhere else, e.g. even another package within this repo.

Reply:
Moreover, the dependency on IPython is currently used, but undeclared (and I would not declare it for Qibolab, since it is not needed for execution)

Reply:
The point of this function is to show the model Hamiltonian that the emulator is running on. Though the additional dependencies are jupyter notebook specific, I would think that the function itself is still useful, at least for users/developers who would want to modify the model. I can simplify it to print in command line an uglier version, or generate an output txt file with the same information perhaps when the emulator platform is initialized? I'm ok to have this in another package but at this point it seems a little awkward to have another package just to host this one function.

Reply:
Hopefully, the model should have a data representation, and that's enough for Qibolab.

You can create a `src/qibolab_visualization` package (eventually, I'd move there even the pulses plotting).

Reply:
Fixed in 0d5129a784f1ed495f971e27fa778fe4aab854c6. Also created a minimal version of `src/qibolab_visualization`, sorry I am not familiar with setting up packages so it's scuffed and I am sure it is missing a lot of settings... but it works.

Reply:
@jykhoo1987 I saw https://github.com/qiboteam/qibolab_visualization, for the time being we don't need anything else. Just add your content in there, and manually install from `git` whenever you need it, using:
```sh
pip install 'git+https://github.com/qiboteam/qibolab_visualization'
```